src/lib/mina_base/snapp_command.ml

```diff
@@ -937,79 +937,14 @@ module Payload = struct
         , Two_proved.Digested.Checked.t )
         Poly.t
 
-      let to_input (t : t) =
-        let open Random_oracle_input in
-        let b = field in
-        let ( ! ) = Impl.run_checked in
-        let inner
-            ({ second_starts_empty
-             ; second_ends_empty
-             ; token_id
-             ; other_fee_payer_opt
-             ; one
-             ; two
-             } :
-              _ Inner.t) ~f1 ~f2 =
-          let p f { Party.Predicated.Poly.body; predicate } =
-            List.reduce_exn ~f:append [ b body; f predicate ]
-          in
-          List.reduce_exn ~f:append
-            [ bitstring [ second_starts_empty; second_ends_empty ]
-            ; !(Token_id.Checked.to_input token_id)
-            ; Snapp_basic.Flagged_option.(
-                to_input' ~f:Other_fee_payer.Payload.Checked.to_input
-                  other_fee_payer_opt)
-            ; p f1 one
-            ; p f2 two
-            ]
-        in
-        let nonce x = !(Account_nonce.Checked.to_input x) in
-        match t with
-        | Zero_proved r ->
-            inner r ~f1:nonce ~f2:nonce
-        | One_proved r ->
-            inner r ~f1:field ~f2:nonce
-        | Two_proved r ->
-            inner r ~f1:field ~f2:field
+      let to_input (_ : t) = failwith "unused"
```

Comment:
Why unused?

Reply:
This code is going to be deleted in the hard-fork